semconv-incubating/src/main/java/io/opentelemetry/semconv/incubating/MessagingIncubatingAttributes.java

```diff
@@ -343,15 +473,31 @@ public final class MessagingIncubatingAttributes {
    * Describes the <a
    * href="https://learn.microsoft.com/azure/service-bus-messaging/message-transfers-locks-settlement#peeklock">settlement
    * type</a>.
+   *
+   * <p>
+   *
+   * @deprecated Replaced by {@code messaging.servicebus.destination.subscription_name}.
    */
   public static final AttributeKey<String> MESSAGING_SERVICEBUS_DISPOSITION_STATUS =
       stringKey("messaging.servicebus.disposition_status");
 
-  /** Number of deliveries that have been attempted for this message. */
+  /**
+   * Number of deliveries that have been attempted for this message.
+   *
+   * <p>
+   *
+   * @deprecated Replaced by {@code messaging.servicebus.destination.subscription_name}.
```

Comment:
I don't think that this is actually deprecated. Applies to several other attributes in this file as well. https://github.com/open-telemetry/semantic-conventions/blob/main/docs/messaging/azure-messaging.md#span-attributes